packages/core/echo/echo-schema/src/automerge/automerge-db.ts

```diff
@@ -182,10 +185,17 @@ export class AutomergeDb {
 
     invariant(isAutomergeObject(obj));
     invariant(!this._objects.has(obj.id));
+
+    const spaceDocHandle = this.automerge.repo.create<SpaceDoc>();
+    this._initDocAccess(spaceDocHandle);
+
     this._objects.set(obj.id, obj);
+    spaceDocHandle.on('change', this._onDocumentUpdate.bind(this));
+    this._linkObjectDocument(obj, spaceDocHandle);
```

Comment:
Right now, because we don't have an object index, this will break queries (they rely on all objects being in memory). The interim solution is to have a hardcoded list of object types that go into linked storage. For now this is only TextObject that gets put into linked storage, other types go into inline map.